dynamic/info.go

```diff
@@ -35,6 +36,7 @@ func providerInfo(ctx context.Context, p run.Provider, value parameterize.Value)
 		Name:        p.Name(),
 		Version:     p.Version(),
 		Description: "A Pulumi provider dynamically bridged from " + p.Name() + ".",
+		Publisher:   "Pulumi",
```

Comment:
I feel like this should be the author/publisher of the bridged provider.

Our docs describe publisher as: "The name of the person or organization that authored and published the package."

Given that there's not really any authoring here it should use the original author imo

Reply:
We are the "publisher" of the bridged provider (if it is published), but not the author. It's a grey area IMO.